_includes/newsletter_link.html

```diff
@@ -1,6 +1,6 @@
 <footer class="slice newsletter">
     <div class="container">
-        <a class="newsletter-link" rel="nofollow noreferrer" href="http://eepurl.com/cOuOIf">
+        <a class="newsletter-link no-link-style tracked-link" data-tracking="newsletter"  rel="nofollow noreferrer" href="http://eepurl.com/cOuOIf">
```

Comment:
can you remove the `no-link-style` class
this is a styling class not a tracking one